lib/hccrawler.js

```diff
@@ -8,7 +8,7 @@ const each = require('lodash/each');
 const includes = require('lodash/includes');
 const isString = require('lodash/isString');
 const isArray = require('lodash/isArray');
-const request = require('request-promise');
+const rp = require('request-promise');
```

Comment:
to avoid name collision with https://github.com/yujiosaka/headless-chrome-crawler/pull/223/files#diff-1b8c3610c5c4374df25c5dd5f30362edR495